CefSharp.MinimalExample.Wpf/Binding/Behaviors/BindHtmlAttachedBehavior.cs

```diff
@@ -40,8 +40,8 @@ private static void OnHtmlChanged(DependencyObject d, DependencyPropertyChangedE
             {
                 return;
             }
-
-            (d as ChromiumWebBrowser)?.LoadHtml((string)e.NewValue, "http://test/page");
+            
+            (d as LoadHtmlBehavior)?.AssociatedObject.LoadHtml((string)e.NewValue, "about:blank");
```

Comment:
`HtmlUrl` property isn't being used. Also best not o use `about:blank`, as it's a special case. You'd be effectively registering a `ResourceHandler` that gets loaded every time `about:blank` is required.